src/InstallationValidator.Core/Services/BatchOutputLoader.cs

```diff
@@ -11,15 +14,95 @@ public interface IBatchOutputLoader
 
    public class BatchOutputLoader : IBatchOutputLoader
    {
+      private readonly IDimensionFactory _dimensionFactory;
+
+      public BatchOutputLoader(IDimensionFactory dimensionFactory)
+      {
+         _dimensionFactory = dimensionFactory;
+      }
+
       public BatchSimulationExport LoadFrom(string fileName)
       {
          var serializer = new JsonSerializer();
 
-         using (var sr = new StreamReader(fileName))
-         using (var reader = new JsonTextReader(sr))
+         try
+         {
+            using (var sr = new StreamReader(fileName))
+            using (var reader = new JsonTextReader(sr))
+            {
+               return serializer.Deserialize<BatchSimulationExport>(reader);
+            }
+         }
+         catch (Exception e)
+         {
+            //try to load the old way and convert to new way. Temporary for v11
+            //required to load again since the reader was consumed
+            using (var sr = new StreamReader(fileName))
+            using (var reader = new JsonTextReader(sr))
+            {
+               var oldBatchSimulationExport = serializer.Deserialize<OldBatchSimulationExport>(reader);
+               return mapToBatchSimulationExport(oldBatchSimulationExport);
+            }
+         }
+      }
+
+      private BatchSimulationExport mapToBatchSimulationExport(OldBatchSimulationExport oldBatchSimulationExport)
+      {
+         //convert to batch simulation export
+         var batchSimulationExport = new BatchSimulationExport
+         {
+            Name = oldBatchSimulationExport.Name,
+            AbsTol = oldBatchSimulationExport.AbsTol,
+            RelTol = oldBatchSimulationExport.RelTol,
+            Times = new BatchValues
+            {
+               Unit = "h",
+               Values = oldBatchSimulationExport.Times,
+               Dimension = "Time"
+            },
+            ParameterValues = oldBatchSimulationExport.ParameterValues,
+         };
+
+         foreach (var outputValues in oldBatchSimulationExport.OutputValues)
          {
-            return serializer.Deserialize<BatchSimulationExport>(reader);
+            var dimension = _dimensionFactory.Dimension(outputValues.Dimension);
+            batchSimulationExport.OutputValues.Add(new BatchOutputValues
+            {
+               Path = outputValues.Path,
+               Values = outputValues.Values,
+               Dimension = outputValues.Dimension,
+               Unit = dimension.DefaultUnitName,
```

Comment:
I think this is how it was exported but we will see when we compare :)

Reply:
it was definitely NOT in base unit since time is exported in hr

Reply:
ok. let's merge and see what happens :)